pages/api/slide/[id].ts

```diff
@@ -0,0 +1,28 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ErrorResponse } from "types/ErrorResponse";
+import connectDB from "../utils/mongoose";
+import { del } from "./delete";
+import { get } from "./get";
+import { patch } from "./patch";
+
+const index = async (
+    req: NextApiRequest,
+    res: NextApiResponse<ErrorResponse>,
+) => {
+    switch (req.method) {
```

Comment:
What's the difference between `/module/id/slide/id` and `/slide/id` routes (can we have slides that are not associated with a module)?